.github/workflows/Pack_Release.yml

```
@@ -0,0 +1,136 @@
+name: Pack Releases
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 版本号 (不带v)
+        required: true
+
+jobs:
+  auto-pack-releases:
+    runs-on: windows-latest
+
+    steps:
+      - name: 检出代码
+        uses: actions/checkout@v4
+
+      - name: 配置 Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.x
+      - name: 安装依赖
+        # 环境配置的依赖文件中含有打包所需依赖
+        run: |
```

```suggestion
      - name: 安装依赖
        env:
          PYTHONIOENCODING: utf-8
        # 环境配置的依赖文件中含有打包所需依赖
        run: |
```

日志过长，请见 https://github.com/fjwxzde/Fufu_Tools/actions/runs/12670138438/job/35309156447#step:4:19